README.md

```diff
@@ -1,11 +1,41 @@
-# Autoscale AWS Library
+# FortiGate Autoscale on AWS
 
-This is an intial readme template.
+The implementation for FortiGate Autoscale using [Fortinet Autoscale](https://github.com/fortinet/autoscale-core) on Amazon AWS platfrom.
+
+This project provides a multi-group Auto Scaling functionality for Fortinet FortiGate EC2 instances to form an HA cluster with failover protection.
+
+This project has the following features:
+
+1. Multi-group Hybrid Licensing models:
```

Comment:
```suggestion
1. Multi-group Licensing models:
```

Reply:
as discussed, we will keep the Hybrid so we do no take this suggestion.